packages/v4/src/content/design-guidelines/components/truncate/truncate.md

```diff
@@ -0,0 +1,54 @@
+---
+id: Truncate 
+section: components
+---
+
+A **truncate** is a tool used to shorten numeric and non-numeric character strings, typically when the string overflows its container. 
+
+## Elements
+* **Content**: text string to truncate
+* **Position**: where the text will be truncated from, ie. front, middle, or end
+* **Tooltip** hover: displayes full-text string 
```

Comment:
You highlighted "Tooltip" but forgot the hover haha should be "Tooltip hover" all bold